README.md

```diff
@@ -40,7 +40,7 @@ As of today KubeVirt can be used to declaratively
  * Stop a VM
  * Delete a VM
 
-Example:
+Click here to see an example video:
```

Comment:
maybe `Click the image below to see an example video` ?

Reply:
sure